### ISSUE TITLE
Change Port

### ISSUE THREAD

**Issue by Donk119** [SemkoDev/nelson.mon#6]
Hello is it possible to change the port 3000 to another? This would be nice because port 3000 is allready in use on my system.

Thanks

**Comment by nazarimilad** [comment]
+1 . I suggest to add an option to change the listening port, like `nelson.mon --port 3200`

**Comment by romansemko** [comment]
Yeah, we can do that. As I said, this package was a fast solution put together in 20 minutes. We are working on a better one. 👍
